docs/content/users/usage/cms-settings.md

```diff
@@ -36,16 +36,70 @@ This environment variable is set `true` by default in DDEV’s environment, and
 
 ### Drupal Specifics
 
-* **Settings Files**: By default, DDEV will create settings files for your project that make it “just work” out of the box. It creates a `sites/default/settings.ddev.php` and adds an include in `sites/default/settings.php` to bring that in. There are guards to prevent the `settings.ddev.php` from being active when the project is not running under DDEV, but it still should not be checked in and is gitignored.
-* **Database requirements for Drupal 9.5+ +**:
-    * Using MySQL or MariaDB, Drupal requires `SET GLOBAL TRANSACTION ISOLATION LEVEL READ COMMITTED` and DDEV does this for you on [`ddev start`](../usage/commands.md#start).
-    * Using PostgreSQL, Drupal requires the`pg_trm` extension. DDEV creates this extension automatically for you on `ddev start`.
-* **Twig Debugging**: With the default Drupal configuration, it’s very difficult to debug Twig templates; you need to use `development.services.yml` instead of `services.yml`. Add this line in your `settings.php` or `settings.local.php`. See discussion at [drupal.org](https://www.drupal.org/forum/support/module-development-and-code-questions/2019-09-02/ddev-twig-debugging) and the Drupal documentation.
+#### Settings Files
+
+By default, DDEV will create settings files for your project that make it “just work” out of the box. It creates a `sites/default/settings.ddev.php` and adds an include in `sites/default/settings.php` to bring that in. There are guards to prevent the `settings.ddev.php` from being active when the project is not running under DDEV, but it still should not be checked in and is gitignored.
+
+#### Database requirements for Drupal 9.5+
+
+* Using MySQL or MariaDB, Drupal requires `SET GLOBAL TRANSACTION ISOLATION LEVEL READ COMMITTED` and DDEV does this for you on [`ddev start`](../usage/commands.md#start).
+* Using PostgreSQL, Drupal requires the`pg_trm` extension. DDEV creates this extension automatically for you on `ddev start`.
+
+#### Twig Debugging
+
+With the default Drupal configuration, it’s very difficult to debug Twig templates; you need to use `development.services.yml` instead of `services.yml`. Add this line in your `settings.php` or `settings.local.php`. See discussion at [drupal.org](https://www.drupal.org/forum/support/module-development-and-code-questions/2019-09-02/ddev-twig-debugging) and the Drupal documentation.
 
 ```php
 $settings['container_yamls'][] = DRUPAL_ROOT . '/sites/development.services.yml';
 ```
 
+#### Multisite
+
+* See [DDEV-Local Drupal 8 Multisite Recipe] (<https://github.com/drud/ddev-contrib/tree/master/recipes/drupal8-multisite>) to start
+* update the following files
+
+1. each `site/{site_name}/settings.php`
+
+     ```php
+     /**
+     * ddev local-dev environments will have $databases (and other settings)
+     * set by an auto-generated file. Make alterations here for this site
+     * in a multisite environment.
+     */
+    elseif (getenv('IS_DDEV_PROJECT') == 'true') {
```

Comment:
This starts with elseif, I'm sure that means something is wrong here.

I also took the liberty of rebasing and fiddling with the flow here, and think I've fixed the spellcheck problem; looks to me like a bug in the spellchecker.